engine/lite/deploymentManager/src/main/scala/pl/touk/nussknacker/lite/manager/LiteDeploymentManagerProvider.scala

```diff
@@ -2,30 +2,48 @@ package pl.touk.nussknacker.lite.manager
 
 import com.typesafe.config.Config
 import pl.touk.nussknacker.engine.api.component.AdditionalPropertyConfig
+import pl.touk.nussknacker.engine.api.definition.{LiteralIntegerValidator, MandatoryParameterValidator, MinimalNumberValidator, StringParameterEditor}
 import pl.touk.nussknacker.engine.api.process.ProcessName
 import pl.touk.nussknacker.engine.api.{LiteStreamMetaData, RequestResponseMetaData}
 import pl.touk.nussknacker.engine.requestresponse.api.openapi.RequestResponseOpenApiSettings
 import pl.touk.nussknacker.engine.{DeploymentManagerProvider, TypeSpecificInitialData}
 
 trait LiteDeploymentManagerProvider extends DeploymentManagerProvider {
 
-  private val streamingInitialMetData = TypeSpecificInitialData(LiteStreamMetaData(Some(1)))
+  private val streamingInitialMetaData = TypeSpecificInitialData(LiteStreamMetaData(Some(1)))
 
   override def typeSpecificInitialData(config: Config): TypeSpecificInitialData = {
     forMode(config)(
-      streamingInitialMetData,
+      streamingInitialMetaData,
       (scenarioName: ProcessName, _: String) => RequestResponseMetaData(Some(defaultRequestResponseSlug(scenarioName, config)))
     )
   }
 
+  private val parallelismConfig: (String, AdditionalPropertyConfig) = "parallelism" ->
+    AdditionalPropertyConfig(
+      defaultValue = None,
+      editor = Some(StringParameterEditor),
+      validators = Some(List(MandatoryParameterValidator, LiteralIntegerValidator, MinimalNumberValidator(1))),
```

Comment:
Like for flink - we rather should make all properties optional and use designer default values in that cases